src/main/Thumbnail.tsx

```diff
@@ -209,15 +213,19 @@ const ThumbnailTableRow: React.FC<{
   track: Track,
   index: number,
   inputRefs: any,
+  generateRef: any,
   generate: any,
   upload: any,
   uploadCallback: any,
   discard: any,
-}> = ({track, index, inputRefs, generate, upload, uploadCallback, discard}) => {
+}> = ({track, index, inputRefs, generateRef, generate, upload, uploadCallback, discard}) => {
 
   const { t } = useTranslation()
   const theme = useTheme();
 
+  // The "+40" comes from padding that is not included in the "getWidth" function
+  const videoWidth = generateRef ? generateRef.getWidth() + 40 : undefined
```

Comment:
Would using `offsetWidth` instead of `clientWidth` remove the need to add 40 manually?

Reply:
It did not

Reply:
Sad :shrug: